Editor/Windows/AssetPaletteWindowEntryPanel.cs

```diff
@@ -230,72 +231,157 @@ private void DrawEntriesPanel()
 
             float containerWidth = Mathf.Floor(EditorGUIUtility.currentViewWidth) - FolderPanelWidth;
 
-            const float padding = 2;
-            int entrySize = Mathf.RoundToInt(Mathf.Lerp(EntrySizeMin, EntrySizeMax, ZoomLevel));
+            GUILayout.Space(Padding);
+
+            bool didClickASpecificEntry;
+            if (ZoomLevel == 0)
```

Comment:
Maybe make a nice property for this, like `ShouldDrawListView` or something.

There's also a `ZoomLevel != 0` down there somewhere so it would be best to have that logic in one place.

Reply:
That's a good call yep